src/hooks/usePermissions.ts

```
@@ -0,0 +1,288 @@
+              setUserPermissions(null)
+            }
+          } else {
+            setError('Erro ao inicializar permissões')
+            setUserPermissions(null)
+          }
+        } else {
+          setError(response.error.userMessage)
+          setUserPermissions(null)
+        }
+      }
+    } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)